src/main/java/cz/fi/muni/pa165/esports/dao/MatchRecordDao.java

```diff
@@ -2,19 +2,20 @@
 
 
 import cz.fi.muni.pa165.esports.entity.MatchRecord;
+import cz.fi.muni.pa165.esports.entity.Player;
 
 import java.util.List;
 
-/*
-    @Author Elena Álvarez
-*/
+/**
+ @Author Elena Álvarez
+ */
 public interface MatchRecordDao {
```

Comment:
Please add javadoc for the interface, you can take a look [here](https://github.com/kaprijela/PA165-esports/pull/38/commits/f6804b0c70600fb6a5bf29927daa888315b2f2e9) to see what I have in mind :)

Reply:
perfect Im on it